.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.7', '3.0', '3.1']#, '3.2']
+        ruby: ['2.7', '3.0', '3.1']
```

Comment:
You can leave the commented version in. I'm about to enable 3.2 in my next commit and it'll just be a conflict at that point anyway. ;)

Reply:
nevermind. This is the only thing that would change, so I'm just merging :) Thanks again!